zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1482,6 +1482,9 @@ components:
           type: string
         order:
           type: string
+      required:
+        - field
+        - order
```

Comment:
💭 Just to check in: do we want to allow sorting with a default behavior, i.e. allow to just define the sort fields and by default sort ASC if no order is provided?

Reply:
So far, the current implementation is expecting the `order` to be not null.
I think it makes sense to default to `asc` for better DevX
I will update the api spec 👍

Reply:
done!